devlab/tests/test_verify_dst_functionality.py

```diff
@@ -0,0 +1,318 @@
+import tests.config as config
+import tests.functional_test as functional_test
+import time
+import pprint
+from fabric.api import run, sudo, local, settings
+from neutronclient.common.exceptions import NeutronClientException
+from novaclient.exceptions import Forbidden
+from novaclient.exceptions import ClientException as NovaClientBaseException
+from cinderclient.exceptions import ClientException as CinderClientBaseException
+from keystoneclient.exceptions import \
+    ClientException as KeystoneClientBaseException
+from glanceclient.common.exceptions import \
+    BaseException as GlanceClientBaseException
+
+
+TIMEOUT = 300
+TEST_TENANT_NAME = 'tenant4'
+TEST_VM_NAME = 'VMtoVerifyDstCloudFunc'
+
+CINDER_VOLUME_CHK_CMDS = ['mkdir /tmp/test',
+                          '/usr/sbin/mkfs.ext2 /dev/vdb',
+                          'mount /dev/vdb /tmp/test',
```

Comment:
Does this command work? The OS assigns the volume to the next available device, in case of cirros /dev/vda